core/src/components/input-password-toggle/input-password-toggle.ionic.scss

```diff
@@ -3,10 +3,25 @@
 
 // Ionic Input Toggle Password
 // --------------------------------------------------
-:host ion-button.button-has-icon-only {
-  --padding-end: inherit;
-  --color: #{globals.$ionic-color-neutral-500};
+:host ion-button.in-input-password-toggle {
+  --color: #{globals.$ionic-color-neutral-800};
   --margin-end: calc(-1 * var(--padding-end));
+  --padding-end: inherit;
 
   @include globals.margin(0, var(--margin-end), 0, 0);
+
+  /* 
+  * '--size' variable is being set at the input.ionic.scss
+  *     • It depends on the size of the input
+  */
+  width: var(--size);
+  height: var(--size);
+
+  ion-icon {
+    /* 
+    * '--icon-size' variable is being set at the input.ionic.scss
+    *     • It depends on the size of the input
+    */
```

Comment:
```suggestion
```

I don't think we need these comments here if we set the default in this component